ui/components/app/transaction-list/transaction-list.test.js

```diff
@@ -1,23 +1,102 @@
+import {
+  MetaMetricsEventCategory,
+  MetaMetricsEventLinkType,
+  MetaMetricsEventName,
+} from '../../../../shared/constants/metametrics';
+import {
+  MULTICHAIN_NETWORK_BLOCK_EXPLORER_URL_MAP,
+  MULTICHAIN_TOKEN_IMAGE_MAP,
+  MultichainNetworks,
+} from '../../../../shared/constants/multichain/networks';
 import TransactionList from './transaction-list.component';
 
-const render = () => {
-  const store = configureStore({
-    metamask: {
-      ...mockState.metamask,
-      transactions: [],
+const defaultState = {
+  metamask: {
+    ...mockState.metamask,
+    transactions: [],
+  },
+};
+
+const btcState = {
+  metamask: {
+    ...mockState.metamask,
+    internalAccounts: {
+      ...mockState.metamask.internalAccounts,
+      accounts: {
+        ...mockState.metamask.internalAccounts.accounts,
+        [MOCK_ACCOUNT_BIP122_P2WPKH.id]: MOCK_ACCOUNT_BIP122_P2WPKH,
+      },
+      selectedAccount: MOCK_ACCOUNT_BIP122_P2WPKH.id,
     },
-  });
-  return renderWithProvider(<TransactionList />, store);
+    selectedAddress: MOCK_ACCOUNT_BIP122_P2WPKH.address,
+    providerConfig: {
+      chainId: MultichainNetworks.BITCOIN_TESTNET,
+      rpcUrl: '',
+      ticker: 'BTC',
+      nickname: 'Bitcoin (testnet)',
+      id: 'btc-testnet',
+      type: 'rpc',
+      rpcPrefs: {
+        imageUrl: MULTICHAIN_TOKEN_IMAGE_MAP[MultichainNetworks.BITCOIN],
+        blockExplorerUrl:
+          MULTICHAIN_NETWORK_BLOCK_EXPLORER_URL_MAP[
+            MultichainNetworks.BITCOIN_TESTNET
+          ],
+      },
+      isAddressCompatible: true,
+    },
+    transactions: [],
+  },
+};
+
+const mockTrackEvent = jest.fn();
+
+const render = (state = defaultState) => {
+  const store = configureStore(state);
+  return renderWithProvider(
+    <MetaMetricsContext.Provider value={mockTrackEvent}>
+      <TransactionList />
+    </MetaMetricsContext.Provider>,
+    store,
+  );
 };
 
 describe('TransactionList', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
   it('renders TransactionList component and shows You have no transactions text', () => {
     render();
     expect(screen.getByText('You have no transactions')).toBeInTheDocument();
   });
+
+  it('renders TransactionList component and shows Bitcoin activity is not supported text', () => {
+    render(btcState);
+
+    expect(
+      screen.getByText('Bitcoin activity is not supported'),
+    ).toBeInTheDocument();
+    const viewOnExplorerBtn = screen.getByRole('button', {
+      name: 'View on block explorer',
+    });
+    expect(viewOnExplorerBtn).toBeInTheDocument();
+    fireEvent.click(viewOnExplorerBtn);
+    expect(mockTrackEvent).toHaveBeenCalledWith({
+      event: MetaMetricsEventName.ExternalLinkClicked,
+      category: MetaMetricsEventCategory.Navigation,
+      properties: {
+        link_type: MetaMetricsEventLinkType.AccountTracker,
+        location: 'Activity Tab',
+        url_domain: 'blockstream.info',
```

Comment:
We could probably use this:

```ts
const blockExplorerDomain = new Url(
  MULTICHAIN_NETWORK_BLOCK_EXPLORER_URL_MAP[MultichainNetworks.BITCOIN]
).host;
```

and use it like:

```ts
  url_domain: blockExplorerDomain,
```

Reply:
Addressed here [4a2ec33](https://github.com/MetaMask/metamask-extension/pull/25642/commits/4a2ec33555fa300ac9dd2e6d3087e37da678860b)